### DISCUSSION TITLE
Python Black Formatter Config

### DISCUSSION THREAD

**Discussion by bpafoshizle** [super-linter/super-linter#1279] [Q&A]
Can someone share the configuration for Python black formatter? I can't seem to get the format that superlinter is checking for in my pipeline to match what black and VS code are doing to format my code locally. I've tried VS code settings and pyproject.toml settings, tweaking the line length, etc. Github actions black formatter wants this code to be on one line, but my local wants it to be multiline:

```
-                section.find("span", class_="item-date"),
-                "string",
-                datetime.now(),
+                section.find("span", class_="item-date"), "string", datetime.now(),
```

**Comment by GaboFDC** [comment]
The config file we use for each linter are under `TEMPLATES/` directory.
Specifically for black, we use a emty one, so I belive the default rules will  apply.
You can customize, to use your specific confics (the ones you use locally), by adding the file, and setting the `PYTHON_BLACK_CONFIG_FILE` env var pointing to your config file.

**Reply by bpafoshizle** [reply]
Thanks @GaboFDC I was wondering about the significance of the empty black file in the TEMPLATES/ directory. I will try to a specific config as you've suggested. It's strange because the behavior has been different even when I was not using a config at all. I am on a different version of Python, though, I believe. Looks like 3.9 is used in super-linter, while I am on 3.8.
